core/coreunix/add.go

```diff
@@ -251,12 +253,14 @@ func (adder *Adder) outputDirs(path string, fsn mfs.FSNode) error {
 	}
 }
 
-// Add builds a merkledag from the a reader, pinning all objects to the local
-// datastore. Returns a key representing the root node.
+// Builds a merkledag node from a reader, adds it to the blockstore,
```

Comment:
Fixed this and the other, and I now have golint installed which looks useful :+1: